src/main/scala/example/Program.scala

```diff
@@ -0,0 +1,76 @@
+package example
+
+import java.time.LocalDate
+import java.time.LocalDate._
+
+import example.Impl._
+import better.files._
+import example.Model.JobResults
+
+import scala.util.Try
+import scala.util.chaining._
+import Model.OptionPickler._
+import com.typesafe.scalalogging.LazyLogging
+
+/**
+ * Main business logic.
+ *
+ * 1. Calculate resume pointer from object.bookmark file if it exists otherwise from beginning of time
+ * 2. Break export into monthly chunks from bookmark until now
+ * 3. Start AQUA export job for each chunk in sequence
+ * 4. Keep checking if the job is done
+ * 5. Download chunk to object-YYYY-MM-DD.csv
+ * 6. Append lines from the chunk to aggregate file object.csv
+ * 7. Once all chunks have downloaded write object.success file
```

Comment:
Could these comments be used to structure the code? ie. name the code like this so that can see main business logic by looking at code